docs/dev_guides/manipulators.md

```diff
@@ -0,0 +1,64 @@
+# Manipulators
+
+Manipulators are UI elements attached to napari layer data to aid in positioning and orienting layer data elements 
+being displayed on the canvas. Manipulators can translate and rotate along/around specified axes.
+
+## Coordinate system
+The manipulator coordinate system is defined with an affine transformation relative to the layer data coordinate system.
+The manipulator transformation is defined as a translation and a 3x3 transformation matrix (rotation and scale). The 
+translation is stored in the `manipulator.centroid` property and the transformation is stored in the `manipulator.
+rot_mat` property. The rotation is applied before the translation. 
+
+## Translators
+Translators are the UI element on the manipulator responsible for translating the manipulator. When the user 
+clicks on a translator and drags it, the manipulator is translated by the drag vector component parallel to the 
+translator direction. 
+
+Translators are defined by unit vectors pointing in the direction of translation. The unit vectors are stored in the 
+`_initial_translator_normals` property as an (n x 3) numpy array for n translators. One translator will be created 
+for each unit vector in `initial_translator_normals`.
+
+### Translator drag callback
+When a translator is dragged, the following callbacks are executed:
+
+1. `_pre_drag()`:This is called after the translator or rotator is clicked. This is typically used to set up for the 
+   drag callback.
+2. `_while_translator_drag()`: This is called during the drag and is typically used to update layer or manipulator 
+   attributes in response to the translator drag. 
+3. `_on_click_cleanup`: This is called after the drag has finished. This is generally used to clean up any 
+   variables that were set during the drag or pre drag callbacks.
```

Comment:
drag -> drag -> click feels a bit inconsistent, area for improvement moving forwards - otherwise very clear, awesome!

Reply:
Good point. I had been thinking it was the clean up of the `on_click` callback, but the user/developer doesn't directly interact with `on_click`, so your suggestion makes more sense.